packages/nuxt/layouts/default.vue

```diff
@@ -20,9 +20,10 @@
       <template #navigation>
         <SfHeaderNavigationItem
             v-for="(item, i) in navigation"
-            :key="i">
-          <SfLink :link="item.link">{{item.name}}</SfLink>
-        </SfHeaderNavigationItem>
+            :key="i"
```

Comment:
suggestion: I know this is a copy-paste, but I think using only index as key is not really descriptive and can cause updates issue, maybe we should change it to something like: ```sf-header-navigation-item-${i}```? WDYT?